src/EPR.Accreditation.Facade/Services/AccreditationService.cs

```diff
@@ -287,5 +287,33 @@ public async Task<HasOverseasAgentDto> GetHasOverseasAgent(Guid id)
 
             await _httpAccreditationService.UpdateAccreditation(id, accreditation);
         }
+
+        /// <summary>
+        /// Gets PRN tonnage data for the given accreditation.
+        /// </summary>
+        /// <param name="accreditationExternalId">Accreditation Id.</param>
+        /// <returns>DTO containing PRN tonnage data.</returns>
+        public async Task<PrnTonnesPlannedDto> GetPrnTonnesPlanned(Guid accreditationExternalId)
+        {
+            var accreditation = await _httpAccreditationService.GetAccreditation(accreditationExternalId);
```

Comment:
This is like the review we talked about last week: we shouldn't need to get the accreditation. Just create an empty one with the properties you need and send that across to the back end - it *should* be able to deal with it